spec/functions/attributes_spec.rb

```diff
@@ -11,10 +11,9 @@
     {
       :kernel => 'Linux',
       :os     => {
-        :family => 'Debian',
-        :name => 'Debian'
```

Comment:
This was a Bug, and only did not appear because `$::osfamily` was used in the PP-Files.